client/services/FeedbackGeneratorService.js

```diff
@@ -92,13 +92,14 @@ tie.factory('FeedbackGeneratorService', [
     };
 
     var _getCorrectnessTestFeedback = function(
-      correctnessTest, observedOutput) {
+      correctnessTest, observedOutput, stdOutput) {
```

Comment:
Let's just call it stdout. There are two types of "output" here and I think it makes sense to think of the second one as just a stdout feed.